src/commands/install.js

```diff
@@ -95,6 +95,8 @@ export default async function install(args: Args, opts: Opts) {
 
   logger.log('[2/2] Linking workspace dependencies...');
 
+  await fs.mkdirp(project.pkg.nodeModulesBin);
```

Comment:
Should we create this directory or should we make a `fs.readdirSafe()` that returns an empty array when the directory does not exist?

Reply:
I figured an empty directory wouldnt be a dangerous thing to have, happy to update to readdirSafe though, since we already wrap all the fs functions as it is

Reply:
I added the function but didnt end up actually needing it. I think some of the work you did with bin linking recently must have fixed it?